Makefile

```diff
@@ -517,6 +517,13 @@ test: generate manifests build-cli-mocks ## Run tests
 	# Test capabilities
 	$(MAKE) test -C capabilities
 
+    # Test plugins
+	cd cmd/cli/plugin/feature && $(GO) test ./... -coverprofile cover.out
+	cd cmd/cli/plugin/login && $(GO) test ./... -coverprofile cover.out
+	cd cmd/cli/plugin/package && $(GO) test ./... -coverprofile cover.out
```

Comment:
For the `plugin/package`, the tests in the `test` directory were designed to be run as integration tests. You can see where they are run at https://github.com/vmware-tanzu/tanzu-framework/blob/74a09a79587ee32ab6b1d6cea41ef91af9e39d66/.github/workflows/package_integration_test.yaml#L101-L102

I think for Codecov, only the unit tests are needed, which can be run with,

```
cd cmd/cli/plugin/package/ && $(GO) test . -coverprofile cover.out
cd kctrl && $(GO) test . -coverprofile cover.out
cd ../openapischema && $(GO) test . -coverprofile cover.out
```

Unfortunately, the percent test coverage is low, but I think that problem needs to be addressed in a separate PR.

```
❯ pwd
~/go/src/github.com/vmware-tanzu/tanzu-framework/cmd/cli/plugin/package
❯ go test . -cover
ok  	github.com/vmware-tanzu/tanzu-framework/cmd/cli/plugin/package	0.391s	coverage: 30.6% of statements
❯ cd kctrl && go test . -cover
ok  	github.com/vmware-tanzu/tanzu-framework/cmd/cli/plugin/package/kctrl	0.401s	coverage: 53.7% of statements
❯ cd ../openapischema &&  go test . -cover
ok  	github.com/vmware-tanzu/tanzu-framework/cmd/cli/plugin/package/openapischema	0.220s	coverage: 90.3% of statements
```

Reply:
Thanks for catching this @codegold79 !
I've update the PR accordingly.  However, to be more future-proof (in case an directory is added for example), I used `go list ./...` to select the tests to run.  I took the idea from https://github.com/vmware-tanzu/tanzu-framework/blob/74a09a79587ee32ab6b1d6cea41ef91af9e39d66/Makefile#L495

Is that ok with you?

Reply:
> Is that ok with you?

Totally--I had no idea about `go list`. I think it's great and I'm glad you found it.